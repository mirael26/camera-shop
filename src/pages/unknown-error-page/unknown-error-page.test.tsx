import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import UnknownErrorPage from './unknown-error-page';

test('UnknownErrorPage renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<UnknownErrorPage/>);
  expect(asFragment()).toMatchSnapshot();
});
