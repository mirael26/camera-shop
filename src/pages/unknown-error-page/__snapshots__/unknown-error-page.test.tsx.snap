// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnknownErrorPage renders correctly 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
    data-testid="not-found-page"
  >
    <main>
      <div
        class="page-content"
      >
        <section
          class="unknown-error"
          data-testid="unknown-error"
        >
          <div
            class="container"
          >
            <h1
              class="title title--h2 unknown-error__title"
            >
              Что-то пошло не так :(
            </h1>
            <a
              class="btn btn--purple"
              href="/"
            >
              Вернутся на главную
            </a>
          </div>
        </section>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
